elections/2021-SC-EU/enduser.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Knative Steering Committee End User Seat"
+linkTitle: "Steering Committee End User Seat"
+weight: 30
+type: "docs"
+---
+
+# 2021 Knative Steering Committee End User Seat
+
+## Purpose
+
+This document describes the process for filling the End User Representative
+for Knative Steering Committee. The newly elected representative will serve a
+one (1) year term.
+
+## Background
+
+In 2021 we introduced one end user seat to bring more diversity into the SC
+representation which is to be elected by the SC once the SC election was
+completed.
+
+## Eligibility
+
+In order to be considered for End User seat, the company the user is
+representing must be listed in the [Adopters](../ADOPTERS.md) file.
```

Comment:
Until we decided to donate func to Knative, I was considering running for the End User seat, since I assumed consuming the APIs was "end user" activity. It's a weird line to draw and not entirely clear where it should be drawn.

Reply:
@jberkus 
>The Adopters File contains companies who are definitely vendors, as well. Do we need to break out an End User section of that file first?

Good idea to add a column for to put "Vendor" or "End User"

Reply:
This was never addressed.   It's kind of a critical fail for the end user "election", since it makes the end user seat no different from the contributor seats.

Reply:
Agreed, being an entry in Adopters.md is not a sufficient criteria for being considered to be an enduser. (e.g. all vendors are mentioned in Adopters.md as well). 

Anything we can do to fix ? // cc @knative/steering-committee

Reply:
Before riding a merged PR, I opened an issue at https://github.com/knative/community/issues/917, so let's continue over there.